analogwp-templates.php

```diff
@@ -126,19 +126,44 @@ function analog_fail_load() {
 	if ( ! function_exists( 'get_plugins' ) ) {
```

Comment:
Why are we still requiring `get_plugins` given that was the cause of all the issue/workaround here?

Reply:
Moved the conditional statement inside the else block.